addons/web/static/src/js/views/pivot/pivot_view.js

```diff
@@ -67,9 +67,11 @@ odoo.define('web.PivotView', function (require) {
 
             this.arch.children.forEach(field => {
                 let name = field.attrs.name;
-
-                // Remove invisible fields from the measures
-                if (field.attrs.invisible && py.eval(field.attrs.invisible)) {
+                // Remove invisible fields from the measures if not in additionalMeasures
```

Comment:
We also would like to separate measures from groupables.
So the final code should look like the following:
```js
if (field.attrs.invisible && py.eval(field.attrs.invisible)) {
    if (name in groupableFields) {
        delete groupableFields[name];
    }
    if (!additionalMeasures.includes(name)) {
        delete measures[name];
        return;
    }
}
```